### DISCUSSION TITLE
Hoymiles HMS-800W-2T

### DISCUSSION THREAD

**Discussion by jnicke** [tbnobody/OpenDTU#1449] [Q&A]
Hallo,
funktioniert dieser Wechselrichter auch mit openDTU?

Bei der Eingabe der SN im openDTU ist die SN eine Stelle zu lang. Fehler in openDTU oder funzt das nur bei dem ohne W

Danke und Gruß
Jürgen

**Comment by stefan123t** [comment]
Nein bis jetzt noch nicht. Und es ist auch noch nicht ganz klar ob / wie es funktionieren könnte. 

Aber Du kannst ja mal bei https://github.com/henkwiedig/Hoymiles-DTU-Proto oder in #1430 bzw. https://github.com/lumapu/ahoy/issues/1189 vorbeischauen, da sind schon ein paar Interessierte dabei das zu reverse-engineeren. Die protobuf Nachrichten der Cloud API und der SDK App API sind wohl schon lesbar.

**Reply by jnicke** [reply]
ok. vielen Dank. Dann nehm ich den ohne W

**Comment by SunboX** [comment]
Ich habe bei meiner Fertig-Stecker-Anlage leider auch genau diesen dabei und möchte ihn irgendwie ins Home Assistant integrieren. Ich hänge mich mal an diesen Thread dran.

**Reply by 2000ede** [reply]
Hallo @SunboX , mit HomeAssistant scheint es nun zu gehen. Siehe Youtube. U.a. Solar Gladbeck.

Ich selber möchte es mit Tasmota haben. Mir fehlen aber noch Infos

**Reply by elsbrock** [reply]
geht problemlos mit https://github.com/DennisOSRM/hms-mqtt-publisher/tree/main/ha-hoymiles-wifi-addon
